lib/ex_unit/lib/ex_unit.ex

```diff
@@ -282,6 +282,12 @@ defmodule ExUnit do
       what tests get run. This is typically used by Mix to filter which tests
       should run;
 
+    * `:rand_algorithm` - algorithm to be used when generating the test seed.
+      Available algorithms can be found in Erlang's
+      [`:rand`](https://www.erlang.org/doc/man/rand.html) documentation (see
+      [`:rand.builting_arg/0`](https://www.erlang.org/doc/man/rand.html#type-builtin_alg)).
+      Defaults to `:exsss`;
```

Comment:
```suggestion
      Available since v1.16.0. Before v1.16.0, the algorithm was hard-coded to
      `:exs1024`. On Elixir v1.16.0 and after, the default changed to `:exsss`;
```